libs/api/domains/cms/src/lib/search/importers/article.service.ts

```diff
@@ -50,6 +64,9 @@ export class ArticleSyncService implements CmsSyncProvider<IArticle> {
             relatedArticles: (relatedArticles.length
               ? relatedArticles
               : undefined) as IArticleFields['relatedArticles'],
+            subArticles: (subArticles.length
+              ? subArticles
+              : undefined) as IArticleFields['subArticles'],
```

Comment:
This might cause the article document to contain alot of unused data since articles only use title in display, consider implementing a more aggressive prune here to patch this until nested resolvers or other permanent solutions are implemented

Reply:
You are right. It only needs the slug/url and the title.